sdk/program/src/sysvar/epoch_rewards.rs

```diff
@@ -0,0 +1,139 @@
+//! Epoch rewards for current epoch
+//!
+//! The _epoch rewards_ sysvar provide access to the [`EpochRewards`] type,
+//! which tracks the progress of epoch rewards distribution. It includes the
+//!   - total rewards for the current epoch, in lamports
+//!   - distributed rewards for the current epoch, in lamports, so far
+//!   - distribution completed block height, i.e. after this block height for distribution of all
+//!     staking rewards for the current epoch will be completed should have been completed.
```

Comment:
nit: maybe copy what's on the struct?
```suggestion
//!   - distribution completed block height, i.e. distribution of all staking rewards for the current
//!     epoch will be completed at this block height
```

Reply:
```suggestion
//!   - block height at which staking rewards for the current epoch will be completed
```
Here's my suggestion.